### PR TITLE
added scss for footer

### PR DESCRIPTION
**Pull request by Aleksandra-Wiktoria-Bury**
Added basic styling for the footer

### CODE REVIEW

File: client/src/assets/scss/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+footer {
+  position: fixed;
+  bottom: 0;
+  background-color: rgb(90, 86, 86);
```
Comment (SebastianSalasDCI) [inline client/src/assets/scss/_footer.scss]: Color is nice, but we select something more bright

**Review by SebastianSalasDCI — CHANGES_REQUESTED** [review]
Please look at the color of the footer

File: client/src/assets/scss/_footer.scss (hunk 1 above)
Comment (Aleksandra-Wiktoria-Bury) [reply client/src/assets/scss/_footer.scss]: agree, let's do yellow

**Review by Aleksandra-Wiktoria-Bury — COMMENTED** [review]
colour to be changed

**Review by SebastianSalasDCI — APPROVED** [review]
The color is good now

**Pull request merged**